tests/manage/pv_services/test_regression_project_delete_leaves_wrong_number_of_pvs_and_rbds.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf8 -*-
+
+import logging
+import textwrap
+import yaml
+
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier3
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_pvc_dict(name):
+    """
+    Creates new copy of PVC dict to tweak and use.
+    """
+    template = textwrap.dedent(f"""
```

Comment:
I still think it would be better to utilize the existing way of creating the resources using the template yamls and modification using helper functions. The outcome is the same and code duplication is saved

Reply:
It makes sense to have general pvc dict fixture in root `conftest.py`, which could be done after this is merged.